paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/AddPaymentMethodRequirement.kt

```diff
@@ -48,12 +50,24 @@ internal enum class AddPaymentMethodRequirement {
     /** Requires a valid us bank verification method. */
     ValidUsBankVerificationMethod {
         override fun isMetBy(metadata: PaymentMethodMetadata): Boolean {
-            val pmo = metadata.stripeIntent.getPaymentMethodOptions()[PaymentMethod.Type.USBankAccount.code]
+            val pmo = metadata.stripeIntent.getPaymentMethodOptions()[USBankAccount.code]
             val verificationMethod = (pmo as? Map<*, *>)?.get("verification_method") as? String
             val supportsVerificationMethod = verificationMethod in setOf("instant", "automatic")
             val isDeferred = metadata.stripeIntent.clientSecret == null
             return supportsVerificationMethod || isDeferred
         }
+    },
+
+    /** Requires that Instant Debits are possible for this transaction. */
+    InstantDebits {
+        override fun isMetBy(metadata: PaymentMethodMetadata): Boolean {
+            val paymentMethodTypes = metadata.stripeIntent.paymentMethodTypes
+            val validTypes = Link.code in paymentMethodTypes && USBankAccount.code !in paymentMethodTypes
```

Comment:
You don't need the `Link.code in paymentMethodTypes` here. That'll happen automatically.